modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -22,7 +22,39 @@ If multiple policies meet this criteria, the policies are sorted by alphabetical
 
 .Procedure
 
-. Create a `MigrationPolicy` object as in the following example:
+. Edit the VM object to which you want to apply a live migration policy, and add the corresponding VM labels.
+
+.. Open the YAML configuration of the resource:
++
+[source,terminal]
+----
+$ oc edit vm <vm_name>
+----
+
+.. Adjust the required label values in the `.spec.template.metadata.labels` section of the configuration. For example, to mark the VM as a `production` VM for the purposes of migration policies, add the `kubevirt.io/environment: production` line:
++
+[source,yaml, subs="+quotes"]
+----
+apiVersion: migrations.kubevirt.io/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: <vm-name>
+  namespace: default
+  labels:
+    app: my-app
+    environment: production
+spec:
+  template:
+    metadata:
+      labels:
+        kubevirt.io/domain: <vm-name>
+        kubevirt.io/size: large
+        kubevirt.io/environment: production
```

Comment:
This might not be a complete YAML example, so consider ending the YAML with:

> # ...

This syntax aligns with the following guideline: 

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#yaml-formatting

Reply:
You're right, it's not complete, will indicate as much in the codeblock